### ISSUE TITLE
Feature request: Respect camera orientation

### ISSUE THREAD

**Issue by sgofferj** [thegoodhen/AprilTools#18]
At the moment, the tag is always laid flat on the ground in Blender, even if it was e.g. pinned at a wall in the video. If possible, it would be very nice, if the correct orientation would be imported into the Blender scene.

Awesome work, by the way!

**Comment by BohdanOpyr** [comment]
You can just parent the camera to the marker and then rotate & move the marker. The only thing that can be added is the auto-parenter in the import addon!

**Comment by Echolink50** [comment]
Seems like that should work but it doesn't or I am doing something wrong. The marker is stable but getting the orientation to the ground seems difficult. I tried parenting the camera to the marker and rotating the marker 90 degrees on the X then raising the marker along the Z axis to the measured real world height. I get a stable marker and jittery objects like maybe they are too high or low in relation to the ground plane

**Comment by BohdanOpyr** [comment]
The problem might be that the marker is too lowres on the video, then it won't give a good result anyway. Try printing a bigger marker maybe or holding a camera closer to it... I don't know, maybe that's actually a parenting problem and we need such a feature...

**Comment by Echolink50** [comment]
The tag was 20" ( +500mm) printed on 12 pages. I used a smaller tag on one page also with the same result. The plane that covers the tag was stable. Anything placed on that wall plane was stable.  Trying to get anything on the floor/ground plane would he jittery

**Comment by sgofferj** [comment]
Depending on your workflow, your keyframes might be 1 frame off. Image sequences usually start with 0000 but Blender animations with frame number one. The plugin doesn't correct that so you might need to go to the dope sheet and move all keyframes of the camera one to the right.

**Comment by Echolink50** [comment]
Yes, I am aware. I just use the frame offset but moving the key frames is probably better. I get a great result with the tag for the ground, but something about the wall tag doesn't quite get the floor right.
